Makefile

```diff
@@ -63,7 +63,7 @@ clean:
 	rm -rf tools.timestamp $(MANPAGES)
 
 test:
-	@$(GPGME_ENV) GO111MODULE="on" go test $(BUILDFLAGS) -cover ./...
+	@PATH="$(PATH):$(shell pwd)/pkg/docker/config/testdata" $(GPGME_ENV) GO111MODULE="on" go test $(BUILDFLAGS) -cover ./...
```

Comment:
Please don’t change global configuration for a single test like this. See e.g. how `XDG_RUNTIME_DIR` is handled in other tests, including the comments about `t.Parallel()`.

Reply:
Fixed, implemented like `XDG_RUNTIME_DIR` in the unit test itself.